src/fonts/liberation-sans-bold.css

```diff
@@ -0,0 +1,4 @@
+@font-face {
```

Comment:
Shouldn't fonts be put inside for example static/fonts folder and type-face in some global index.css file?

Reply:
Moved fonts and css to public/fonts directory